devtools/src/strategy-explorer/se-recipe-view.html

```diff
@@ -44,14 +44,16 @@
       }
     </style>
     <template is='dom-if' if='{{shownRecipe}}'>
-      <div class='recipe-box'>{{strategyString}}<div hidden$="[[isDescriptionEmpty(shownRecipe.description)]]" class="description">
+      <div class='recipe-box'>{{strategyString}}<!--
```

Comment:
Is this comment here to prevent a newline from being rendered?

Reply:
Yup, it would cause an additional line break where one is not necessary.